content/algorithms/dag/index.md

```diff
@@ -0,0 +1,146 @@
+---
+jupytext:
+  notebook_metadata_filter: all
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+language_info:
+  codemirror_mode:
+    name: ipython
+    version: 3
+  file_extension: .py
+  mimetype: text/x-python
+  name: python
+  nbconvert_exporter: python
+  pygments_lexer: ipython3
+  version: 3.9.2
+---
+
+# Directed Acyclic Graphs
+
+In this tutorial, we'll explore the algorithms related to directed acyclic graph
+(or a "dag" as it is sometimes called) implemented in networkx under `networkx/algorithms/dag.py`.
```

Comment:
By @rossbar: Maybe at the beginning of the notebook show how to construct DAGs and non-DAGs to help illustrate the concept using NetworkX. Some code cells to create the graph and quickly show properties.